Apps/Src/Task_VoltTempMonitor.c

```diff
@@ -56,7 +57,7 @@ void Task_VoltTempMonitor(void *p_arg) {
         for (int i = 0; i < NUM_BATTERY_MODULES; i++){ //send all battery module voltage data
             CanPayload.idx = i;
             int voltage = Voltage_GetModuleMillivoltage(i);
-            CanData.f = (float)voltage/1000; //send data in volts
+            memcpy(&CanData.w, &voltage, sizeof(CanData.w));
```

Comment:
same question as before.

Reply:
Same reason as before, but we could probably make `voltage` unsigned